src/canister_tests/src/api/internet_identity.rs

```diff
@@ -312,25 +312,34 @@ pub mod compat {
     use super::*;
     use candid::{CandidType, Deserialize};
     use internet_identity_interface::internet_identity::types::{
-        ActiveAnchorCounter, ActiveAnchorStatistics, AnchorNumber, ArchiveInfo,
-        DomainActiveAnchorCounter,
+        DeviceData, DeviceRegistrationInfo, DeviceWithUsage,
```

Comment:
FYI: it's cut off by the default diff, but this is replacing the old (no longer needed) compat shim with the new one for metadata.